doc/api/path.md

```diff
@@ -2,31 +2,86 @@
 
     Stability: 2 - Stable
 
-This module contains utilities for handling and transforming file
-paths. The file system is not consulted to check whether paths are valid.
+The `path` module provides utilities for working with file paths. It can be
+accessed using:
 
-Use `require('path')` to use this module.  The following methods are provided:
+```js
+const path = require('path');
+```
+
+The file system is not consulted to determine if paths are valid.
+
+## Windows vs. posix
+
+The default operation of the `path` module varies based on the operating system
+on which a Node.js application is running. Specifically, when running on the
+Windows operating system, the `path` module will assume that Window-style
+paths are being used.
+
+For example, using the `path.basename()` function with the Windows file path
+`C:\temp\myfile.html`, will yield different results when running on posix than
+when run on Windows:
+
+On posix:
+
+```js
+path.basename('C:\\temp\\myfile.html');
+  // returns 'C:\temp\myfile.html'
+```
+
+On Windows:
+
+```js
+path.basename('C:\\temp\\myfile.html');
+  // returns 'myfile.html'
+```
+
+To achieve consistent results when working with Windows file paths on any
+operating system, use [`path.win32`][]:
```

Comment:
It may/may not be helpful to add the word "namespace" after `path.win32` here. Similarly for the `path.posix` reference below.